lib/card_generator.rb

```diff
@@ -0,0 +1,13 @@
+class CardGenerator
+attr_reader :file
+  def initialize(filename)
+    @filename = filename
+  require "pry"; binding.pry
```

Comment:
Nice job starting on iteration 4! When you have time later on, it might be good to come back and finish this out.